ontopy/ontology.py

```diff
@@ -936,6 +940,18 @@ def save(
         # pylint: disable=too-many-statements,too-many-branches
         # pylint: disable=too-many-locals,arguments-renamed,invalid-name
 
+        if namespaces is None:
+            namespaces = {}
+        default_namespaces = {
+            "": self.base_iri,
+            "locn": "http://www.w3.org/ns/locn#",
+            "swrl": "http://www.w3.org/2003/11/swrl#",
+            "bibo": "http://purl.org/ontology/bibo/",
```

Comment:
What is the reason for this choice? We have other, more often typically used namespaces.

Reply:
OK, then add this information as a comment above.

Reply:
done